content/docs/setup/kubernetes/upgrade/index.md

```diff
@@ -268,3 +277,67 @@ The script automates the following operations:
 
 1. The script deletes the previous RBAC configuration custom resource after applying the cluster RBAC
    configuration successfully.
+
+## Special one-time upgrade procedure for early adopters
+
+If you have previously used Helm with Tiller to upgrade an
+Istio installation from `0.8` to `1.0`, you will need to use the following instructions
+to upgrade from Istio `1.0` to `1.1`. This is because of various Tiller bugs in the `0.8` and `1.0`
+release timeframes. If you simply use `helm upgrade` at this point, it will result in CRD corruption and
+will leave your Istio installation in an inoperable state.
+
+To upgrade, you will need to download the release images for Istio `0.8.0`, `1.0.6`, and `1.1.0`, and then
+run the following commands:
+
+```
+cd istio-0.8.0
+kubectl create -f install/kubernetes/helm/helm-service-account.yaml
+helm init --service-account tiller
+helm install install/kubernetes/helm/istio --name istio --namespace istio-system --set mixer.enabled=false
+kubectl apply -f - <<EOF
```

Comment:
this is a validation mechanism - we can delete this or use it for validation and request at every step folks dump thiis.